### ISSUE TITLE
?Variation model redundancy Nonsense/Missense vs #Molecular_change hash

### ISSUE THREAD

**Issue by Paul-Davis** [WormBase/website#4569]
We can probably remove redundant top level tags in the ?Variation class

```
?Variation
//
                       Phenotype_not_observed ?Phenotype XREF Not_in_Variation #Phenotype_info  //added by Wen to separate Not phenotype from real phenotypes 
-                       Nonsense UNIQUE Amber_UAG Text #Evidence
-                                       Ochre_UAA Text #Evidence
-                                       Opal_UGA Text #Evidence
-                       Missense Text #Evidence                // text fields stored details of codon change
                       Silent Text #Evidence
//
```
As these aren't used and the data has been moved into the #Molecular_change hash

Paul

**Comment by epaule** [comment]
I do believe, I left them in for curatorial reasons (to separate them from the automatically derived ones), but @mt3 might know more.

**Comment by Paul-Davis** [comment]
@epaule If so they aren't used? yes will let @tuli decide if they can go.

**Comment by kyook** [comment]
Just to be clear you are only talking about these lines :

-                       Nonsense UNIQUE Amber_UAG Text #Evidence
-                                       Ochre_UAA Text #Evidence
-                                       Opal_UGA Text #Evidence
-                       Missense Text #Evidence                // text
fields stored


~~~~
Karen Yook

Curator
WormBase Caltech
Tel: 626.395.8325
e-mail: kyook@caltech.edu
e-mail: karen@wormbase.org
skype name: wbkaren


On Wed, Feb 10, 2016 at 8:28 AM, Paul Davis <notifications@github.com>
wrote:

> @epaule <https://github.com/epaule> If so they aren't used? yes will let
> @tuli <https://github.com/tuli> decide if they can go.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/4569#issuecomment-182461977>.
>

**Comment by tuli** [comment]
IIRC there was a period of time when I wanted to ensure that all of those
curations (from prior to #molecular_change) had been successfully
transferred (either automatically or manually) to #molecular_change.

If you're telling me that none of those tags are used then I'm happy that
this transfer has happened and so they can be removed.

Silent and Readthrough are also not used.

I will look at the Splice_site and Frameshift ones.

M.A.

On 10 February 2016 at 16:28, Paul Davis <notifications@github.com> wrote:

> @epaule <https://github.com/epaule> If so they aren't used? yes will let
> @tuli <https://github.com/tuli> decide if they can go.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/4569#issuecomment-182461977>.
>



-- 
~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125

**Comment by Paul-Davis** [comment]
@kyook ummmm...no

I am talking about all lines with a - symbol.

Hope that's alright.

P

**Comment by Paul-Davis** [comment]
@tuli ah there is still a very small amount of data

21 - Splice_site
15 - Frameshift

If these were migrated we could remove the whole stack.

**Comment by tuli** [comment]
Yes, I already noted those were there. It will be good to remove the whole
stack. Nice tidy models. I'll let you know.

On 12 February 2016 at 09:53, Paul Davis <notifications@github.com> wrote:

> @tuli <https://github.com/tuli> ah there is still a very small amount of
> data
>
> 21 - Splice_site
> 15 - Frameshift
>
> If these were migrated we could remove the whole stack.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/4569#issuecomment-183256788>.
>



-- 
~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125

**Comment by Paul-Davis** [comment]
@tuli thanks

**Comment by tuli** [comment]
Splice_sites all cleared up in Geneace for WS253 so this top level tag can
be removed.

On 12 February 2016 at 10:05, Paul Davis <notifications@github.com> wrote:

> @tuli <https://github.com/tuli> thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/4569#issuecomment-183259240>.
>



-- 
~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125

**Comment by tuli** [comment]
Frameshifts all cleared up in Geneace for WS254 so this top level tag can be removed.
